govuk-prototype-kit.config.json

```diff
@@ -7,7 +7,8 @@
   },
   "pluginDependencies": [{
     "packageName": "govuk-frontend",
-    "minVersion": "3.12.0"
+    "minVersion": "3.12.0",
+    "maxVersion": "4.7.0"
```

Comment:
Thanks for adding a `maxVersion`

As far as I know, these version fields are validated but might only be used at plugin install time to additionally install related dependencies with a minimum or maximum version

Since GOV.UK Frontend is always installed, do you know if these ranges do anything?

Reply:
Doesn't seem like either the `minVersion` or the `maxVersion` do anything? 😞

Reply:
We've updated the description as well, as that’ll be more visible within the 'Mange your prototype' interface.